db/schema.rb

```diff
@@ -162,6 +162,27 @@
     t.index ["framework_question_id"], name: "index_framework_nomis_codes_questions_on_question_id"
   end
 
+  create_table "framework_nomis_mappings", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.jsonb "raw_nomis_mapping", null: false
+    t.string "code", null: false
+    t.string "code_type", null: false
+    t.text "code_description"
+    t.text "comments"
+    t.date "start_date"
+    t.date "end_date"
+    t.date "creation_date"
```

Comment:
Is `creation_date` different from `created_at` ?

Reply:
we wanted to differentiate the `created_at` coming from NOMIS for some resources from the `created_at` datetime table value to avoid confusion, so we went with `creation_date` instead

Reply:
It also ties nicely with `expiry_date`

Reply:
Fair enough 👍